packaging/osx/scripts/postinstall

```diff
@@ -12,19 +12,21 @@ INSTALL_DESTINATION=$2
 chmod -R 755 $INSTALL_DESTINATION
 
 ln -s $INSTALL_DESTINATION/bin/dotnet /usr/local/bin/
-ln -s $INSTALL_DESTINATION/bin/dotnet-compile /usr/local/bin/
 ln -s $INSTALL_DESTINATION/bin/dotnet-build /usr/local/bin/
+ln -s $INSTALL_DESTINATION/bin/dotnet-compile /usr/local/bin/
 ln -s $INSTALL_DESTINATION/bin/dotnet-compile-csc /usr/local/bin/
 ln -s $INSTALL_DESTINATION/bin/dotnet-compile-fsc /usr/local/bin/
+ln -s $INSTALL_DESTINATION/bin/dotnet-compile-native /usr/local/bin/
 ln -s $INSTALL_DESTINATION/bin/dotnet-new /usr/local/bin/
 ln -s $INSTALL_DESTINATION/bin/dotnet-pack /usr/local/bin/
 ln -s $INSTALL_DESTINATION/bin/dotnet-publish /usr/local/bin/
 ln -s $INSTALL_DESTINATION/bin/dotnet-repl /usr/local/bin/
+ln -s $INSTALL_DESTINATION/bin/dotnet-repl-csi /usr/local/bin/
 ln -s $INSTALL_DESTINATION/bin/dotnet-restore /usr/local/bin/
-ln -s $INSTALL_DESTINATION/bin/dotnet-resgen /usr/local/bin/
 ln -s $INSTALL_DESTINATION/bin/dotnet-run /usr/local/bin/
-ln -s $INSTALL_DESTINATION/bin/dotnet-test /usr/local/bin/
 ln -s $INSTALL_DESTINATION/bin/dotnet-dnx /usr/local/bin/
+ln -s $INSTALL_DESTINATION/bin/dotnet-test /usr/local/bin/
```

Comment:
Can we alphabetize these? it's tough to match inventory across the many files in which they're listed without this.